apps/web/lib/d/[link]/[slug]/getServerSideProps.tsx

```diff
@@ -72,6 +79,17 @@ async function getUserPageProps(context: GetServerSidePropsContext) {
   if (!hashedLink) {
     return notFound;
   }
+
+  const isExpired = hashedLink.expiresAt ? new Date(hashedLink.expiresAt).getTime() < Date.now() : false;
```

Comment:
We store in user's timezone EOD converted to UTC, so this checks against that and compares to current UTC time

Reply:
This now checks for utc midnight, no? Shouldn't we also here convert to event type owner's midnight?